tests/conf.js

```diff
@@ -12,7 +12,7 @@ const baseConfig = {
   },
   SELENIUM_PROMISE_MANAGER: false,
   seleniumAddress: 'http://localhost:4444/wd/hub',
-  exclude: ['**/*.wdio-spec.js'],
+  exclude: ['**/*.wdio-spec.js', '**/*.standard-wdio-spec.js'],
```

Comment:
Removing the `.` should get the standard tests too. If we continue the `wdio-spec.js` suffix then we won't have to worry about updating this when we add another config in. 

`exclude: ['**/*wdio-spec.js'],`